apps/src/lib/node/ledger/protocol/transactions/ethereum_events/eth_msgs.rs

```diff
@@ -48,12 +49,8 @@ impl From<MultiSignedEthEvent> for EthMsgUpdate {
 pub struct EthMsg {
     /// The event being stored
     pub body: EthereumEvent,
-    /// The total voting power that's voted for this event across all epochs
-    pub voting_power: FractionalVotingPower,
-    /// The addresses of validators that voted for this event
-    pub seen_by: BTreeSet<Address>,
-    /// Whether this event has been acted on or not
-    pub seen: bool,
+    /// Tracking of votes for this event
+    pub vote_tracking: VoteTracking,
```

Comment:
Suggestions for better naming around `VoteTracking`, `votes`, etc. appreciated!

Reply:
Hope the find / replace wouldn't be too much of a hassle, but I like `VoteTallies` and variations thereof

Reply:
Have updated to `votes::Tally` etc